src/validation.cpp

```diff
@@ -329,7 +329,8 @@ static bool IsCurrentForFeeEstimation(CChainState& active_chainstate) EXCLUSIVE_
     return true;
 }
 
-/* Make mempool consistent after a reorg, by re-adding or recursively erasing
+/**
```

Comment:
nit: doxygen function comments should go with the function declaration. Do you mind moving this to the header file now that the declaration is there?

Reply:
Fixed, thanks!